src/wallet/api/walletApi.ts

```diff
@@ -219,6 +223,47 @@ const walletApi = {
       throw new WalletRequestError('Error requesting auth token.');
     }
   },
+
+  async pushRegister(wallet: HathorWalletServiceWallet, payload: PushRegisterRequestData): Promise<PushRegisterResponseData> {
```

Comment:
From what we discussed, I understand that these `pushRegister`, `pushUpdate` and `pushUnregister` methods should be moved to a separate utils file - possibly on `src/utils/push.ts`, right?

That way the main `src/wallet` module would not be changed.
What do you think?

Reply:
Yes, the `wallet` module is not being affected anymore. This is the module `walletApi`. I see the `walletApi` as a client to talk with the `wallet-service`. What do you think?